src/wallet/test/wallet_tests.cpp

```diff
@@ -765,7 +765,7 @@ BOOST_FIXTURE_TEST_CASE(CreateWallet, TestChain100Setup)
     // being blocked
     wallet = TestLoadWallet(context);
     BOOST_CHECK(rescan_completed);
-    BOOST_CHECK_EQUAL(addtx_count, 2);
+    BOOST_CHECK_EQUAL(addtx_count, 3);
```

Comment:
Agreed.

Reply:
Yeah, I see what you mean. The `TestLoadWallet()` helper function had a call to `postInitProcess()` which meant that the mempool was checked twice. Removing that led to the numbers in the `CreateWallet` test being much more sensible and there seem to be no other negative effects from that change. At least the test isn't failing and I also didn't see any other reason to keep it. I added some comments as well.

Reply:
IMO bumping the numbers was better. :/

Reply:
> IMO bumping the numbers was better. :/

Could you say why you like it better? When the numbers are bumped I feel they don't match what the test is doing and I would need to add more comments to explain why this is the case. Hence it seemed like the better choice to me. Also curious to hear your feedback here @w0xlt and @Sjors :)

Reply:
Seems like it would be undefined behaviour (between modules) to skip `postInitProcess`.

Counting AddToWallet calls might be too low-level to be a good test, but that seems out of scope here IMO.

Reply:
I have no strong feelings about this, as long as it's clear from the code documentation where the transactions are coming from.